myko/src/lib/components/StartActivityButton.svelte

```diff
@@ -39,27 +39,110 @@
 </script>
 
 {#if loading}
-  <div class="loader" />
+  <div class="lds-default">
```

Comment:
This is a perfect candidate to move into a separate Svelte component (so the code can be put together logically in one file and be reused)! 🙂 
To do it: move these div-tags and the related CSS to a file in `components` directory, and replace this with the new tag for the custom component.